R/sample_and_adjust.R

```diff
@@ -45,6 +47,7 @@ sample_and_adjust_by_dose <- function(
   pars_true_i,
   sim_model,
   sim_ruv = NULL,
+  ltbs = FALSE,
```

Comment:
```suggestion
```

see comment below, we can extract this from the model so the user doesn't have to remember to do this.